tools/build.csx

```diff
@@ -1,5 +1,5 @@
-#r "packages/Bullseye.1.1.0-rc.2/lib/netstandard2.0/Bullseye.dll"
-#r "packages/SimpleExec.2.2.0/lib/netstandard2.0/SimpleExec.dll"
+#r "packages/Bullseye/lib/netstandard2.0/Bullseye.dll"
+#r "packages/SimpleExec/lib/netstandard1.3/SimpleExec.dll"

```

Comment:
@FakeItEasy/owners was the change back to `netstandard1.3` intentional? The restoration graph is much lighter if you use `netstandard2.0`.

Reply:
@adamralph no, it's not intentional. I must have done it when I resolved a conflict with https://github.com/FakeItEasy/FakeItEasy/pull/1409